iphone/Classes/TiUIiOSNavWindowProxy.m

```diff
@@ -181,6 +181,11 @@ - (void)windowClosing:(TiWindowProxy *)window animated:(BOOL)animated
 
 - (void)navigationController:(UINavigationController *)navigationController willShowViewController:(UIViewController *)viewController animated:(BOOL)animated
 {
+  if ([TiUtils isIOSVersionOrGreater:@"11.2"]) {
+    navigationController.navigationBar.tintAdjustmentMode = UIViewTintAdjustmentModeNormal;
+    navigationController.navigationBar.tintAdjustmentMode = UIViewTintAdjustmentModeAutomatic;
```

Comment:
Both? That doesn‘t seem righr.

Reply:
Yes. If you see the discussion at https://stackoverflow.com/questions/47754472/ios-uinavigationbar-button-remains-faded-after-segue-back, the same is suggested.

Reply:
Weird, but valid. Can you file a radar for this as well?

Reply:
Yes. I'll file bug to apple.